css/style.css

```diff
@@ -1,11 +1,227 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+header {
+  display: flex;
+  padding: 40px;
+  margin-left: 100px;
+}
+
+/* Main Image*/
+.main_image {
+  margin-bottom: 600px;
+  position: relative;
+  top: -70px;
+}
+
+*/*Top Left Karma Logo*/
+.header__logo {
+  height: 50px;
+  margin-right: 900px;
+}
+
+/*Top Right Navigation Links*/
+a:hover {
+  color: orange;
+}
+
+.navigation__list {
+  display: flex;
+  position: relative;
+  bottom: 25px;
+}
+
+.navigation__item {
+  color: gray;
+  padding: 0;
+  display: flex;
+}
+
+.navigation__link {
+  color: var(--grey-dark);
+  font-weight: 600;
+  text-decoration: none;
+  margin: 30px;
+}
+
+/*Main*/
+.main_image {
+  background-image: url("../img/first-background.jpg");
+  padding: 200px;
+  background-size: cover;
+}
+.main_intro {
+  text-align: center;
+  padding-top: 100px;
+  color: white;
+  font-size: 50px;
+  letter-spacing: 0.8rem;
+  position: relative;
+  bottom: 100px;
+}
+.wifi_promo {
+  text-align: center;
+  color: white;
+  letter-spacing: 0.5rem;
+  position: relative;
+  bottom: 75px;
+}
+.learn_more_button {
+  color: white;
+  background-color: chocolate;
+  border: none;
+  border-radius: 5px;
+  padding: 20px 40px;
+  margin-left: 650px;
```

Comment:
Why this large left margin? To center elements, rely on Flexbox: apply `display: flex` and `flex-driection: column` on the parent class - in this case `.main_image`.